src/spirit-view/spirit-view.html

```diff
@@ -81,7 +81,7 @@
             },
 
             _next: function() {
-              console.log(this.$.name.valu)
+                console.log(this.$.name.valu)
```

Comment:
Notice that this won't do anything useful, because "name.valu" is undefined. We should just kill this as console noise.